assets/dev/js/frontend/handlers/base.js

```diff
@@ -102,7 +102,7 @@ module.exports = elementorModules.ViewModule.extend( {
 
 					const propName = Object.keys( changedModel.changed )[ 0 ];
 
-					self.onEditSettingsChange( propName, changedModel.changed[ propName ] );
+					self.onEditSettingsChange( propName, changedModel.changed[ propName ], changedModel.attributes.defaultEditRoute );
```

Comment:
Do you have any idea why the active tab index would change because of a content change, i.e. 
`'content' === changedModel.attributes.defaultEditRoute`